core/sender/Sender.cpp

```diff
@@ -1928,6 +1928,55 @@ bool Sender::Send(const std::string& projectName,
         projectName, sourceId, logGroup, config, mergeType, logGroupSize, defaultRegion, filename, context);
 }
 
+int Sender::SendDirectly(std::vector<sls_logs::Log>& logs, Config* config) {
+    uint32_t nowTime = time(NULL);
+    const size_t maxCount = INT32_FLAG(merge_log_count_limit) / 4;
```

Comment:
merge_log_count_limit is used by both Sender::SendDirectly and Aggregator::Add(which is called by Sender::Send) in one processing. The calling relationship is not clear.